### ISSUE TITLE
Funções compostas

### ISSUE THREAD

**Issue by caiadofelipe** [vitor-gabriel/snippets-advpl#3]
Vitor, bom dia.

Tenho algumas funções compostas;

Exemplo: DbselectArea já com dbssek., begin sql já com while.

Vi que vc dividiu os arquivo, podemos fazer um só com essas funcoes compostas.

**Comment by vitor-gabriel** [comment]
Sim, podemos conversar sobre isso, me chama no linkedin se ainda tiver interesse: https://www.linkedin.com/in/vitorgabriel-tech/
